src/App.js

```diff
@@ -17,29 +21,95 @@ What should a cart item look like? 🤔
 console.log(initialStoreItems)
 
 export default function App() {
-  // Setup state here...
+  const [store, setStore] = useState(initialStoreItems)
+  const [cart, setCart] = useState([])
 
+
+  // making a new array of a cart by adding items
+  const addNewItem = (item) => {
+    setCart([...cart, item])
+    console.log("hello".item)
+  }
+
+  // updating items
+  const onIncreaseClick = (target) => {
```

Comment:
you could perhaps move this function completely to the related component rather than passing it down as a prop.